server/DAO/__fixtures__/dates.js

```diff
@@ -0,0 +1,13 @@
+const moment = require('moment');
+
+module.exports = [
+    {
+    value: moment().format("L"),
+    type: "date"
+},
+
+{
+    value: moment().add(1, 'day').format("L"),
+    type: "date"
+}
+]
```

Comment:
Kind of a nit pick but one way you could organize this is by having 
```js
const currentDate =     {
    value: moment().format("L"),
    type: "date"
}

const nextDate = {
    value: moment().add(1, 'day').format("L"),
    type: "date"
}

const previousDate =    {
            "value": moment().add(-1, 'day').format("L"),
            "names": [
              "yesterday"
            ]
          };

module.exports = {
   currentDate,
   nextDate,
   previousDate
}
```

This gives you a handy naming convention when using it in your tests.